google-account-plugin/src/com/google/gct/login/GoogleLogin.java

```diff
@@ -640,7 +656,14 @@ public boolean askYesOrNo(String title, String message) {
       String updatedMessage = message;
       if (message.equals("Are you sure you want to sign out?")) {
         updatedMessage += " This will sign out all logged in users.";
+        for (GoogleLoginMessageExtender messageExtender : messageExtenders) {
+          String additionalLogoutMessage = messageExtender.additionalLogoutMessage();
+          if (!Strings.isNullOrEmpty(additionalLogoutMessage)) {
+            updatedMessage += " " + additionalLogoutMessage;
+          }
+        }
       }
+      updatedMessage = WordUtils.wrap(updatedMessage, WRAP_LENGTH, null, false);
```

Comment:
it helps readability here if you add param docs /* paramName */ as part of each null or boolean flag.